x-pack/plugins/ml/public/application/data_frame_analytics/pages/analytics_creation/components/validation_step/validation_step_details.tsx

```diff
@@ -0,0 +1,99 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import React, { FC } from 'react';
+import { i18n } from '@kbn/i18n';
+import {
+  EuiButtonEmpty,
+  EuiDescriptionList,
+  EuiFlexGroup,
+  EuiFlexItem,
+  EuiIcon,
+  EuiSpacer,
+  EuiText,
+} from '@elastic/eui';
+import { State } from '../../../analytics_management/hooks/use_create_analytics_form/state';
+import { ANALYTICS_STEPS } from '../../page';
+import { ValidationSummary } from './validation_step_wrapper';
+
+export const ValidationStepDetails: FC<{
+  setCurrentStep: React.Dispatch<React.SetStateAction<ANALYTICS_STEPS>>;
+  state: State;
+  validationSummary: ValidationSummary;
+}> = ({ setCurrentStep, state, validationSummary }) => {
+  const { isJobCreated } = state;
+  const detailsFirstCol = [
+    {
+      title: i18n.translate(
+        'xpack.ml.dataframe.analytics.create.validatioinDetails.successfulChecks',
+        {
+          defaultMessage: 'Successful checks',
+        }
+      ),
+      description: (
+        <>
+          <EuiFlexGroup gutterSize="xs" alignItems="center">
+            <EuiFlexItem grow={false}>
+              <EuiText size="s">{validationSummary.success}</EuiText>
+            </EuiFlexItem>
+            <EuiFlexItem grow={false}>
+              <EuiIcon type="check" />
+            </EuiFlexItem>
+          </EuiFlexGroup>
+        </>
+      ),
+    },
+  ];
+  const detailsSecondCol = [
+    {
+      title: i18n.translate('xpack.ml.dataframe.analytics.create.validatioinDetails.warnings', {
+        defaultMessage: 'Warnings',
+      }),
+      description: (
+        <>
+          <EuiFlexGroup gutterSize="xs">
+            <EuiFlexItem grow={false}>
+              <EuiText size="s">{validationSummary.warning}</EuiText>
+            </EuiFlexItem>
+            <EuiFlexItem grow={false}>
+              <EuiIcon type="alert" />
+            </EuiFlexItem>
+          </EuiFlexGroup>
+        </>
+      ),
+    },
+  ];
+  return (
+    <>
+      <EuiFlexGroup style={{ width: '70%' }}>
```

Comment:
Is it possible to rerun the validation before returning to the validation step? For example, I set the training percent too low initially, then went back to the Configuration section and increased it, but the warning didn't disappear until I went back into the Validation step.

![image](https://user-images.githubusercontent.com/7405507/109980665-0dc9e180-7cf8-11eb-971d-0a9ab03ccd4a.png)

Reply:
Adding this to be done in a follow up 👍